RepoDb/RepoDb/ClassProperty.cs

```diff
@@ -272,7 +272,7 @@ public bool Equals(ClassProperty other)
             {
                 return ReferenceEquals(null, objB);
             }
-            return objA?.GetHashCode() == objB?.GetHashCode();
+            return objB?.GetHashCode() == objA.GetHashCode();
```

Comment:
@calumchisholm - all changes that corresponds to this will be soon merge to 'master'.